### PR TITLE
Fetch weather and display weather card or error 

### PR DESCRIPTION
**Pull request by NadzeyaZhukovich**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    const $errorMessage = document.getElementById('errorMessage');
+    const $weatherCard = document.getElementById('weather');
+    const $weatherCardImg = document.getElementById('weatheCurrentDay__img');
+    const $weatheCardCity = document.getElementById('weatheCurrentDay__info-city');
+    const $weatheCardTemperature = document.getElementById('weatheCurrentDay__info-temperature');
+    const $weatheCardDate = document.getElementById('weatheCurrentDay__info-date');
+
+    $searchBtn.addEventListener('click', event => {
+        event.preventDefault();
+        let city = $searchInput.value;
```
Comment (ubcent) [inline app.js]:
```suggestion
        const city = $searchInput.value;
```

File: app.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        let units = $temperatureUnit.options[$temperatureUnit.selectedIndex].value;
```
Comment (ubcent) [inline app.js]:
```suggestion
        const units = $temperatureUnit.options[$temperatureUnit.selectedIndex].value;
```

File: app.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        fetchWeather(city, units);
+    })
+
+    function fetchWeather(city, units) {
+        const url = BASE_URL + 'q=' + city + '&units=' + units + '&appid=' + ApiKeys.openWeatherMapKey;
```
Comment (ubcent) [inline app.js]:
```suggestion
        const url = `${BASE_URL}q=${city}&units=${units}&appid=${ApiKeys.openWeatherMapKey}`;
```

File: app.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        return response;
+    }
+
+    function updateWetherCard(weatherInfo) {
+        const iconCode = weatherInfo.weather[0].icon;
+        $errorMessage.textContent = '';
+        $weatheCardCity.textContent = weatherInfo.name;
+        $weatheCardTemperature.innerHTML = `${weatherInfo.main.temp}&deg;`;
+        $weatheCardDate.textContent = Date.today().toString('MMM dd, dddd');
+        $weatherCardImg.src = 'http://openweathermap.org/img/w/' + iconCode +'.png';
```
Comment (ubcent) [inline app.js]: Шаблонная строка

File: app.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    function handleErrors(response) {
+        if (!response.ok) {
+            throw Error(response.statusText);
+        }
+        return response;
+    }
+
+    function updateWetherCard(weatherInfo) {
```
Comment (ubcent) [inline app.js]: typo

File: app.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        $weatheCardCity.textContent = weatherInfo.name;
+        $weatheCardTemperature.innerHTML = `${weatherInfo.main.temp}&deg;`;
+        $weatheCardDate.textContent = Date.today().toString('MMM dd, dddd');
+        $weatherCardImg.src = 'http://openweathermap.org/img/w/' + iconCode +'.png';
+
+        $weatherCard.style.display = 'block';
+    }
+
+    function showErrorMessage() {
+        $errorMessage.textContent = "Something went wrong";
```
Comment (ubcent) [inline app.js]:
```suggestion
        $errorMessage.textContent = 'Something went wrong';
```

File: app.js (hunk 1 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js (hunk 2 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js (hunk 3 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js (hunk 4 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js (hunk 5 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

File: app.js (hunk 6 above)
Comment (NadzeyaZhukovich) [reply app.js]: Fixed

**Pull request merged**